website/pages/downloads/index.jsx

```diff
@@ -11,7 +11,7 @@ export default function DownloadsPage({ binaryReleases, desktopReleases }) {
   return (
     <div className={styles.root}>
       <HashiHead title={`Downloads | ${productName} by HashiCorp`} />
-      <ProductDownloader
+      <ProductDownloadsPage
```

Comment:
This needs wider changes - the head section and wrapper are included in the page version of the component

Reply:
Thanks, good looking out! I left out the usage of the server helper as we are also fetching for boundary desktop here in parallel.

Reply:
Still can have the wrapping div, styles, and extra head component removed, as those are all repeated 😬

Reply:
Oh wow I think I have a local commit that I didn't push up 🤦 one moment...